src/workers/postUpdated.ts

```diff
@@ -52,6 +52,7 @@ interface Data {
   collections?: string[];
   extra?: {
     keywords?: string[];
+    keywords_native?: string[];
```

Comment:
What exactly is this?
Not sure where it comes from/or why we need it?

Reply:
It comes from Yggdrasil.
`keywords` is generated from the captions, and `keywords_native` comes from video description.

Pulled from the Yggdrasil model:
https://github.com/dailydotdev/yggdrasil/blob/befaa6c92e3e1dc598e6fc2cdb8368c50bbb83f8/services/content_publisher/types.go#L35-L36

Reply:
```suggestion
    keywords_native?: string[];  // Extra keywords field that comes from YouTube videos
```

Could add a comment to it like this?